code/web/src/modules/auth/AuthAdminCheck.js

```diff
@@ -18,6 +18,8 @@ AuthAdminCheck.propTypes = {
 }
 
 // Component State
+// I am not certain what this function is doing??
```

Comment:
Are these the comments from the code annotation branch (maybe it is for BE understanding)? 

- If code annotation - can we remove them before merging this branch?
- If BE reference - I will check with the other BEnders if they want them still 😄